### PR TITLE
Partial fix for schema change errors, get_ByIds update

### LINKED ISSUE #42: get__ByIds doesnt seems to work
Hello ! 
Thanks for the work done here.

I'm using get__ById and get__ByIds methods, but it seems the first one is working fine, but not the last one. One workaround was to create a new field at the end of the table called id with a special formula RECORD_ID(), but it seems wrong (Especially with get_byId working)

Any feedback of this ?

My error : error: "INVALID_FILTER_BY_FORMULA", message: "The formula for filtering records is invalid: Unknown field names: id()

Thanks.

### PR DESCRIPTION
**Pull request by annieyro**
Tried to fix the schema breaking, with some success. TODO: will fill in details tomorrow night after setting up `meepanyar-node` to test with instead 😅

Closes #42 using the proposed solution. I've tested this with `dccentralkitchen-node` and it seems to be fine.

### CODE REVIEW

**Comment by romainquellec** [comment]
👏  @aivantg

File: README.md [hunk 1]
```diff
@@ -284,8 +286,16 @@ Example usage:
...
 ## Notes
 
-This generator was made originally to support Blueprint projects. Learn more about what we do here: https://calblueprint.org/
+This generator was made originally to support Blueprint projects. Learn more about what we do here: <https://calblueprint.org/>
+
+## Known issues
+
+### Schema change errors
+
+In version 1.5.1 and earlier, the generated `airtable.js` would throw errors upon being unable to properly transform the Airtable call internally. It uses the local copy of `schema.js` to perform the transformations, which unfortunately caused client applications to break upon schema changes until the local copy of `schema.js` was updated - i.e the generator was re-run.
```
Comment (aivantg) [inline README.md:300]: Wow this is so professionally written 👏

Comment (aivantg) [reply README.md:300]: Also such a high-leverage change LOL

File: lib/snippets.js [hunk 2]
```diff
@@ -122,7 +122,7 @@ export const get${pluralize(
     )}ByIds = async ( ids, filterByFormula = '', sort = []
 ) => {
   let formula = ${'`OR(${'}ids.reduce(
-    (f, id) => ${"`${f} {ID}='${id}',`"},
+    (f, id) => ${"`${f} RECORD_ID()='${id}',`"},
```
Comment (aivantg) [inline lib/snippets.js:125]: Makes sense + looks good! And you tested this, right?

**Review by aivantg — APPROVED** [review]
Looks good! Let me know if I can help with the todo!

File: README.md (hunk 1 above)
Comment (annieyro) [reply README.md:300]: in this ~~house~~ repo we stan documentation 😂 

I updated it to include the cases I can think of, I'll try to fix the rest later 😔

File: lib/snippets.js (hunk 2 above)
Comment (annieyro) [reply lib/snippets.js:125]: Yes, tested this one (directly from the issue suggestion) (":

**Pull request merged**